addons/l10n_it_edi/models/account_move.py

```diff
@@ -417,6 +417,15 @@ def _l10n_it_edi_grouping_function_total(self, base_line, tax_data):
         skip = tax_data['is_reverse_charge'] or self._l10n_it_edi_is_neg_split_payment(tax_data)
         return not skip
 
+    @api.model
+    def _inverse_negative_quantity_lines(self, base_lines):
```

Comment:
Not sure having a dedicated method is useful, especially if we are only using it once :thinking: 
Maybe we could do something similar as what is already done for `reverse_charge_refund`?
Also, adding a comment explaining why we need to do the inversion might be nice :innocent: